tools/build.sh

```diff
@@ -136,13 +146,5 @@ if [[ "${SECTION_SEARCH}" == "" || "${LANGUAGE}" != "en" ]]; then
   # rm triggers latexmk build even if previous artifacts generated by faulty run of po4a prevent it from running
   rm -f "main_${LANGUAGE}.aux"
 fi
-latexmk -pdflua -file-line-error -halt-on-error -interaction=nonstopmode -shell-escape -recorder "main_${LANGUAGE}.tex"
+latexmk "${ARGS}" -pdflua -file-line-error -halt-on-error -interaction=nonstopmode -shell-escape -recorder "main_${LANGUAGE}.tex"
 ${open} "main_${LANGUAGE}.pdf" &> /dev/null &
-
-# Optimize PDF if it's a single section and ghostscript is available
```

Comment:
not optimizing anymore?

Reply:
It's a copy from the mission book. People would build and share those big files without running ghostscript. It has no use here, and it started to annoy me because of the file flickering.